staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/patch.go

```diff
@@ -1054,7 +1054,7 @@ func applyRetainKeysDirective(original, patch map[string]interface{}, options Me
 // Then, sort them by the relative order in setElementOrder, patch list and live list.
 // The precedence is $setElementOrder > order in patch list > order in live list.
 // This function will delete the item after merging it to prevent process it again in the future.
-// Ref: https://git.k8s.io/community/contributors/design-proposals/preserve-order-in-strategic-merge-patch.md
+// Ref: https://git.k8s.io/community/contributors/design-proposals/cli/preserve-order-in-strategic-merge-patch.md
```

Comment:
ditto

Reply:
I think this will not be used to generate user documentation.  For developer, design proposals is also  appropriate.